### PR TITLE
Move format-string to source

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/37308523/CHI-1934
The snprintf is getting string pointer instead of string literal which inhibits necessary compiler warnings

### CODE REVIEW

**Comment by vbcypherock** [comment]
Task linked: [CHI-1934 Move const format strings to local file from constant_text.c](https://app.clickup.com/t/37308523/CHI-1934)

**Comment by amanCypherock** [comment]
Can we solve this issue by using macro instead of string pointers for format strings?

**Comment by ujjwal-cyph** [comment]
> Can we solve this issue by using macro instead of string pointers for format strings?

Changes ready

File: src/level_four/tap_cards/tasks/write_to_cards_tasks.c [hunk 1]
```diff
@@ -83,7 +83,7 @@ void tap_cards_for_write_flow()
     case TAP_CARD_TWO_FRONTEND:
     case TAP_CARD_THREE_FRONTEND:
     case TAP_CARD_FOUR_FRONTEND:
-        snprintf(display, sizeof(display), ui_text_tap_x_4_cards, ((flow_level.level_four-1)>>1)+1);
+        snprintf(display, sizeof(display), UI_TEX_TAP_CARD, ((flow_level.level_four-1)>>1)+1);
```
Comment (amanCypherock) [inline src/level_four/tap_cards/tasks/write_to_cards_tasks.c:86]: Mis-spelled TEXT

**Review by amanCypherock — CHANGES_REQUESTED** [review]

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**